File: 백준/Bronze/10818. 최소， 최대/최소， 최대.js
const input = require("fs").readFileSync(0).toString().split("\n");
const n = parseInt(input[0]);
const arr = input[1].split(" ").map(number => +number);


let tmp;
let max = arr[0];
let min = arr[0];
for (let i = 0; i < n; i++) {
    if (arr[i] > max) {
        max = arr[i];
    }
    if (arr[i] < min) {
        min = arr[i];
    }
}


console.log(min, max)

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int N = Integer.parseInt(br.readLine());
        
        int[] arr = new int[N];
        StringTokenizer st = new StringTokenizer(br.readLine());
        for(int i=0; i<N; i++){
            arr[i] = Integer.parseInt(st.nextToken());
        }

        int min = 1_000_000;
        int max = -1_000_000;
        for(int value : arr){
            min = Math.min(value, min);
            max = Math.max(value, max);
        }
        System.out.println(min+" "+max);
    }
}
